packages/devtools_server/lib/src/server.dart

```diff
@@ -9,6 +9,7 @@ import 'dart:isolate';
 
 import 'package:args/args.dart';
 import 'package:browser_launcher/browser_launcher.dart';
+import 'package:http_multi_server/http_multi_server.dart';
```

Comment:
Question: what is http_multi_server useful for?

Reply:
It binds to both `localhost` and `127.0.0.1` or the IPv6 equivalent. You also can pass hostname `any` and it will correctly bind to both.